pkg/kubeapiserver/options/authentication_test.go

```diff
@@ -1051,6 +1048,8 @@ jwt:
 }
 
 func TestValidateOIDCOptions(t *testing.T) {
+	featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, utilfeature.DefaultFeatureGate, version.MustParse("1.33"))
```

Comment:
move this inside the testcase t.Run, and only do this in the testcase if structuredAuthenticationConfigEnabled is false

Reply:
done!